test/unittests/evm_eof_test.cpp

```diff
@@ -23,51 +25,51 @@ TEST_P(evm, eof1_execution_with_data_section)
 {
     rev = EVMC_CANCUN;
     // data section contains ret(0, 1)
-    const auto code = eof1_bytecode(mstore8(0, 1) + OP_STOP, ret(0, 1));
+    const auto code = eof1_bytecode(mstore8(0, 1) + OP_STOP, 2, ret(0, 1));
 
     execute(code);
     EXPECT_STATUS(EVMC_SUCCESS);
     EXPECT_EQ(result.output_size, 0);
 }
 
-TEST_P(evm, eof1_pc)
+TEST_P(evm, DISABLED_eof1_pc)
```

Comment:
Why is this disabled and not just removed?

Reply:
@chfast Do you agree to remove it and others below?

Reply:
This and below should be removed

Reply:
All DISABLED tests have been removed.